public/media_engagement/imaqal_sms_traffic.js

```diff
@@ -769,20 +769,23 @@ const update = (data) => {
     var fullDateFormat = d3.timeFormat("%c")	
 
     // Update timestamp of update and reset formatting
-    const timestamp = new Date()
```

Comment:
lets name to be more descriptive e.g `lastDataUpdateTimeStamp` can't think of a suggestion but feel free to change this.

Reply:
lastTimeStampDataUpdated sounds better